codalab/lib/bundle_cli.py

```diff
@@ -1091,7 +1126,7 @@ def do_rm_command(self, args):
         arguments=(
             Commands.Argument('keywords', help='Keywords to search for.', nargs='+'),
             Commands.Argument('-a', '--append', help='Append these bundles to the current worksheet.', action='store_true'),
-            Commands.Argument('-u', '--uuid-only', help='Print only uuids.', action='store_true'),
+            Commands.Argument('-u', '--uuid_only', help='Print only uuids.', action='store_true'),
```

Comment:
Wanted to be consistent everywhere.  I'd actually prefer dashes everywhere, but was held up because the metadata specs automatically generates options with underscores.  We could easily change that though.  Would have to update the docs too.  Let's save for a different PR.